src/space-connector/type.ts

```diff
@@ -2,11 +2,41 @@ export interface SessionTimeoutCallback {
     (): void;
 }
 
+export type ResponseCode = 'APIError' | 'NotFoundError' | 'BadRequestError' | 'AuthenticationError' | 'AuthorizationError'
+
+export interface Response<T> {
+    code: ResponseCode | string;
+    message: string;
+    result: T;
+}
+
 export interface APIInfo {
     path: string;
     methods: Array<string>;
 }
 
+export interface ErrorModel {
+    message: string;
+    code: string;
+}
```

Comment:
```suggestion
interface ErrorModel {
    message: string;
    code: string;
}
```

ServerError.error 에 ErrorModel 이 쓰이므로, 이렇게 정의해 주셨군요.
읽기 편하고 좋습니다!

내부적으로 쓰이기 위한 타입 정의라면 `export`되어야 할 이유가 없는 것 같아요.
외부에서는 `ErrorModel` 를 가져다 쓰기보다는 `ServerError`를 가져다가 쓸 것 같아요.
되려 `ServerError`을 가져다 써야 할 때, 무엇을 가져와야 하나 헷갈릴 것 같아서 `export`를 빼버리면 어떨까 하는 의견입니다 ㅎ

Reply:
하지만 단독으로 충분히 쓰인다면 `export` 하는게 좋아요! 
이 타입정의를 가져다 쓴다는 입장에서 한번 생각해봤어요 ㅎ